kirun-js/src/engine/function/system/array/Max.ts

```diff
@@ -0,0 +1,47 @@
+import { KIRuntimeException } from '../../../exception/KIRuntimeException';
+import { EventResult } from '../../../model/EventResult';
+import { FunctionOutput } from '../../../model/FunctionOutput';
+import { FunctionExecutionParameters } from '../../../runtime/FunctionExecutionParameters';
+import { AbstractArrayFunction } from './AbstractArrayFunction';
+import { Sort } from './Sort';
+
+export class Max extends AbstractArrayFunction {
+    public constructor() {
+        super('Max', [Max.PARAMETER_ARRAY_SOURCE], Max.EVENT_RESULT_ANY);
+    }
```

Comment:
Update the parameters to only primitive source array in both java and ts and in both min and max